Feat change volume model color

adamkglaser opened the pull request:
Adding functionality to change tile grid color when FOV location wanders outside of the tile grid boundary.

- [x] Added init args to control active/inactive visualization of the tile grid
- [x] Added logic into the ```update_opts``` function to toggle the grid color. https://github.com/AllenNeuralDynamics/view/blob/f984035140b22ed9414ea1c2a95a8c8b543e665e/view/widgets/acquisition_widgets/volume_model.py#L374-L390
- [x] Update process calls ```setColor``` to update the color of the wireframe box. The shaded box has no such function, so added an ```updateColor``` function to achieve this behavior. https://github.com/AllenNeuralDynamics/view/blob/f984035140b22ed9414ea1c2a95a8c8b543e665e/view/widgets/miscellaneous_widgets/gl_shaded_box_item.py#L41-L46

micahwoodard commented:
@adamkglaser I modified gl_shaded_box item to allow outlines and also changed where the color change logic happens. Let me know what you think

adamkglaser commented:
> @adamkglaser I modified gl_shaded_box item to allow outlines and also changed where the color change logic happens. Let me know what you think

Thanks this is a cleaner way of doing everything. I am OK to merge when you are.
